### PR TITLE
Add sendy integration

### PR DESCRIPTION
**Pull request by mahmoud**

### CODE REVIEW

**Comment by slaporte** [comment]
Here are the old signup pages we need to swap out in weeklypedia-history:

| lang | mailchimp                | new signup                                       |
| ---- | ------------------------ | ------------------------------------------------ |
| en   | http://eepurl.com/MMlpX  | https://weekly.hatnote.com/archive/en/index.html |
| de   | http://eepurl.com/MMlG9  | https://weekly.hatnote.com/archive/de/index.html |
| fr   | http://eepurl.com/MMmVX  | https://weekly.hatnote.com/archive/fr/index.html |
| ko   | http://eepurl.com/MMm8n  | https://weekly.hatnote.com/archive/ko/index.html |
| et   | http://eepurl.com/MMnlf  | https://weekly.hatnote.com/archive/et/index.html |
| sv   | http://eepurl.com/MMTnP  | https://weekly.hatnote.com/archive/sv/index.html |
| da   | http://eepurl.com/Sko4L  | https://weekly.hatnote.com/archive/da/index.html |
| it   | http://eepurl.com/MQTPb  | https://weekly.hatnote.com/archive/it/index.html |
| ca   | http://eepurl.com/M7HU9  | https://weekly.hatnote.com/archive/ca/index.html |
| eo   | http://eepurl.com/_RE_X  | https://weekly.hatnote.com/archive/eo/index.html |
| es   | http://eepurl.com/br8NNj | https://weekly.hatnote.com/archive/es/index.html |
| fa   | http://eepurl.com/br8TlT | https://weekly.hatnote.com/archive/fa/index.html |
| zh   | http://eepurl.com/bsmGhT | https://weekly.hatnote.com/archive/zh/index.html |
| ur   | http://eepurl.com/bsmG1P | https://weekly.hatnote.com/archive/ur/index.html |
| kn   | http://eepurl.com/buxugf | https://weekly.hatnote.com/archive/kn/index.html |
| lv   | http://eepurl.com/b0V-yj | https://weekly.hatnote.com/archive/lv/index.html |
| el   | http://eepurl.com/b0WcUr | https://weekly.hatnote.com/archive/el/index.html |
| te   | http://eepurl.com/b3EjMn | https://weekly.hatnote.com/archive/te/index.html |
| oc   | http://eepurl.com/dxJIYf | https://weekly.hatnote.com/archive/oc/index.html |
| ru   | http://eepurl.com/dxJJGn | https://weekly.hatnote.com/archive/ru/index.html |
| uk   | http://eepurl.com/dxJKwf | https://weekly.hatnote.com/archive/uk/index.html |

File: weeklypedia/secrets-default.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+	"mc": "...",
+    "sendy_key": "...",
+	"list_ids": {
```
Comment (mahmoud) [inline weeklypedia/secrets-default.json:4]: should these be secret? seem a pretty open one if so...

File: weeklypedia/publish.py [hunk 2]
```diff
@@ -6,34 +6,45 @@
...
+
 
 def get_argparser():
     desc = 'Bake and send Weeklypedia issues. (Please fetch first)'
     prs = ArgumentParser(description=desc)
     prs.add_argument('--lang', default=None)
     prs.add_argument('--bake_all', default=False, action='store_true')
+    prs.add_argument('--nosend', default=False, action='store_true')
     prs.add_argument('--debug', default=DEBUG, action='store_true')
+    prs.add_argument('--mailer', default='mailchimp')
```
Comment (mahmoud) [inline weeklypedia/publish.py:47]: oh is mailchimp stickin around?

File: weeklypedia/publish.py [hunk 3]
```diff
@@ -6,34 +6,45 @@
...
+        result = cur_issue.sendy_send(list_id)
+    return result
+
 
 def get_argparser():
     desc = 'Bake and send Weeklypedia issues. (Please fetch first)'
     prs = ArgumentParser(description=desc)
     prs.add_argument('--lang', default=None)
     prs.add_argument('--bake_all', default=False, action='store_true')
+    prs.add_argument('--nosend', default=False, action='store_true')
```
Comment (mahmoud) [inline weeklypedia/publish.py:45]: --dry-run?

File: weeklypedia/issue_templates/zh_email.txt [hunk 4]
```diff
@@ -40,3 +40,9 @@ The WEEKLYPEDIA: Issue {issue_number}, {date} ({full_lang_name} Wikipedia Editio
 * {stats.bot_count|ci} 個機器人進行 {stats.bot_edits|ci} 次編輯，對於 {stats.bot_titles|ci} 個條目
 
 Learn more about the Weeklypedia Digest: http://weekly.hatnote.com/
+
+This email was sent to [Email]
+
+Unsubscribe: [unsubscribe]
+
+Hatnote · 131 E Empire Street · San Jose, CA 95112 · USA
```
Comment (mahmoud) [inline weeklypedia/issue_templates/zh_email.txt:48]:
```suggestion
Hatnote · 558 North 3rd Street · San Jose, CA 95112 · USA
```

File: weeklypedia/publish.py (hunk 3 above)
Comment (mahmoud) [reply weeklypedia/publish.py:45]: I guess it's not strictly a dry run since it does bake.

File: weeklypedia/secrets-default.json (hunk 1 above)
Comment (slaporte) [reply weeklypedia/secrets-default.json:4]: we won't keep the sendy `list_ids` secret -- only the `sendy_key`.

File: weeklypedia/publish.py (hunk 2 above)
Comment (slaporte) [reply weeklypedia/publish.py]: it's not around anymore!

**Pull request merged**